README.md

```diff
@@ -3106,7 +3106,7 @@ import cats.effect.IO as CatsIO
 import cats.effect.kernel.Outcome.Succeeded
 
 // Note how Cats includes the IO, Async, and Abort[Throwable] effects:
-val a: Int < Cats =
+val a: Int < (Abort[Throwable] & Async) =
```

Comment:
I wonder if we should also use `Abort[Nothing]` like in `ZIOs` and treat failures in cat-effect's execution as panics

Reply:
changed to `Abort[Nothing]`